frontend/webpack.config.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     hints: false
   },
   module: {
-    rules: [isProduction ? {} : {
+    rules: [true || isProduction ? {} : {
```

Comment:
isn't this a debug thing that should be removed ?

Reply:
yes... thx
```suggestion
    rules: [isProduction ? {} : {
```